### DISCUSSION TITLE
Question: problems when connecting to VPN server from DS218+

### DISCUSSION THREAD

**Discussion by dragonflypl** [angristan/openvpn-install#1397] [Help]
Hi,

I have successfully created VPN server with your script. I can connect to it using OpenVPN GUI client on windows. However when I try to connect my NAS: https://www.synology.com/en-global/products/DS218+ , it fails. Perhaps you have encountered this issue before or know what could solve it? 

As for what I tried:
- after googling I realized that NAS does not accept generated ovpn file, in particular I had to remove section, so that it is possible to import the file into NAS:

```
<tls-crypt>
#
# 2048 bit OpenVPN static key
#
-----BEGIN OpenVPN Static key V1-----
.......
-----END OpenVPN Static key V1-----
</tls-crypt>
```
Thanks,
P.

**Comment by angristan** [comment]
> it fails

We will need more detaills

**Comment by randshell** [comment]
You need to remove that from ovpn **and** server configuration. Remember this is insecure.

See https://github.com/angristan/openvpn-install/issues/536#issuecomment-599187829 for how it should be the file.

**Comment by dragonflypl** [comment]
I've reinstalled the server with `tls-auth` instead of `tls-crypt`. Now my ovpn file gets imported, but still NAS cannot connect. How can I check logs of the vpn server ?

**Comment by strabism** [comment]
Hi,

Read this : https://github.com/angristan/openvpn-install#data-channel

I just resolved my problem with that.

Im sure your NAS isnt in Openvpn >=2.4

You should use a CBC ciphers.

I can help you if you want.

**Comment by dragonflypl** [comment]
Thanks, I'll reinstall my server and give it a try

pt., 20 mar 2020 o 00:36 strabism <notifications@github.com> napisał(a):

> Hi,
>
> Read this : https://github.com/angristan/openvpn-install#data-channel
>
> I just resolved my problem with that.
>
> Im sure your NAS isnt in Openvpn >=2.4
>
> You should use a CBC ciphers.
>
> I can help you if you want.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angristan/openvpn-install/issues/558#issuecomment-601464493>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABJRE7G2WBZWJLPFOA7QSALRIKT7VANCNFSM4LDOMZ7Q>
> .
>


-- 
----
Pozdrawiam,
Paweł Waszczyński

**Comment by strabism** [comment]
> Thanks, I'll reinstall my server and give it a try pt., 20 mar 2020 o 00:36 strabism <notifications@github.com> napisał(a):
> […](#)
> Hi, Read this : https://github.com/angristan/openvpn-install#data-channel I just resolved my problem with that. Im sure your NAS isnt in Openvpn >=2.4 You should use a CBC ciphers. I can help you if you want. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#558 (comment)](https://github.com/angristan/openvpn-install/issues/558#issuecomment-601464493)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABJRE7G2WBZWJLPFOA7QSALRIKT7VANCNFSM4LDOMZ7Q> .
> -- ---- Pozdrawiam, Paweł Waszczyński

It's ok?
